core/src/main/java/org/apache/accumulo/core/file/rfile/VisMetricsGatherer.java

```diff
@@ -47,6 +46,7 @@
  */
 public class VisMetricsGatherer
     implements MetricsGatherer<Map<String,ArrayList<VisibilityMetric>>> {
+  static final String KEY_HASH_ALGORITHM = "SHA-256";
```

Comment:
This is an odd one.  I am not sure what we are using the HASH to encrypt here.  If it is just to prevent leaking data then I _think_ it would be OK to use the NON_CRYPTO_USE_HASH_ALGORITHM

Reply:
> This is an odd one. I am not sure what we are using the HASH to encrypt here. If it is just to prevent leaking data then I _think_ it would be OK to use the NON_CRYPTO_USE_HASH_ALGORITHM

@milleruntime I think this was moved here at my request to keep serialization-related constants local to the serialization code, rather than to rely on some global constant, which could have a multitude of uses. I think it's fine here.

Reply:
I don't think this is serializing anything.  I thought this method was only used by the `rfile-info` command. The only use i can find of `printMetircs()` is here: https://github.com/apache/accumulo/blob/6a74b4667e3bd33e34b5262c5dd8ea64167fb657/core/src/main/java/org/apache/accumulo/core/file/rfile/PrintInfo.java#L291

Reply:
By serialization, I simply mean the serial form that we use for this particular display format.

Reply:
That constant can be left inside this class, because it's not being used as some sort of overall crypto strategy that necessitates it be aligned with how the other constant is being used... it's just an arbitrary choice for this particular visibility metric output format (or "serialization algorithm") and relates to nothing else. So, it can/should be a local constant held closer to the rest of that metric serialization algorithm. That's all I mean.

Reply:
Ah I see.  It makes me wonder why are we even bother hashing it at all for display purposes.  Why not just print gobbledygook or just don't print it.